boards/stm32f7508-dk/Kconfig

```diff
@@ -15,6 +15,8 @@ config BOARD_STM32F7508_DK
     # Put defined MCU peripherals here (in alphabetical order)
     select HAS_PERIPH_DMA
     select HAS_PERIPH_ETH
+    select HAS_PERIPH_FMC_SDRAM
```

Comment:
```suggestion
    select HAS_PERIPH_FMC
    select HAS_PERIPH_FMC_SDRAM
```

Reply:
You can squash directly btw

Reply:
Shit, how could I forget this :speak_no_evil: